pmlc/dialect/stripe/autostencil_pass.cc

```diff
@@ -0,0 +1,312 @@
+// Copyright 2019, Intel Corporation
+
+#include "pmlc/dialect/stripe/transcode.h"
+#include "pmlc/dialect/stripe/autostencil_pass.h"
+
+#include <sstream>
+#include <vector>
+
+#include "mlir/IR/Matchers.h"
+#include "mlir/Translation.h"
+
+#include "pmlc/dialect/stripe/analysis.h"
+#include "pmlc/dialect/stripe/dialect.h"
+#include "pmlc/dialect/stripe/ops.h"
+#include "pmlc/dialect/stripe/transforms.h"
+#include "pmlc/dialect/stripe/util.h"
+
+#include "base/util/logging.h"
+
+namespace pmlc {
+namespace dialect {
+namespace stripe {
+
+// Number of tensors for the matrix multiplication
+const size_t numTensors = 3;
```

Comment:
For constants, I think the current style is `kNumTensor`

Reply:
Also, although this is different than our current codebase, I've noticed that LLVM prefers to use `unsigned` over `size_t`